torch/fx/_symbolic_trace.py

```diff
@@ -430,9 +437,18 @@ def call_module(
             value was returned from the ``Module`` invocation.
         """
         module_qualified_name = self.path_of_module(m)
-        if not self.is_leaf_module(m, module_qualified_name):
-            return forward(*args, **kwargs)
-        return self.create_proxy("call_module", module_qualified_name, args, kwargs)
+        with ScopeContextManager(self.scope, Scope(module_qualified_name, type(m))) as _scope:
```

Comment:
I think we might have to check if `self` has `scope` or not, we can skip if it doesn't, to keep BC